reana_db/models.py

```diff
@@ -208,6 +197,18 @@ def initialize_user_quota_limits(self):
                 )
             )
 
+    def get_quota_usage(self):
+        """Get user quota usage information."""
+        return dict(
+            quota=dict(
+                disk=dict(
+                    usage=self.get_user_disk_usage(),
+                    limit=self.get_user_quota_by_type(ResourceType.disk)["limit"],
+                ),
+                cpu=self.get_user_quota_by_type(ResourceType.cpu),
```

Comment:
I guess `disk` dict will look something like this eventually, am I right? Once https://github.com/reanahub/reana-db/issues/88 is implemented

Reply:
Yes, that was my idea, good that we see it similarly :)